### DISCUSSION TITLE
Include tabIndex prop for Combobox.Button

### DISCUSSION THREAD

**Discussion by jordisantamaria** [tailwindlabs/headlessui#3045] [Ideas]
Hi, I want combobox to open on click, so I have to grap combobox trigger into Combobox.Button, but then It can't be selectable by pressing tab from other input because by default it have tabindex="-1".

Could you make it possible to accept a prop called tabIndex to be able to override this behavior?

**Comment by KYPremco** [comment]
I did this by making the `ComboboxButton` absolute, and full width when not open. This will open the options on click, but won't open on tab.

**Comment by jordisantamaria** [comment]
Well, right now I'm getting this behavior by doing this,
![image](https://github.com/tailwindlabs/headlessui/assets/6263700/c648e87a-23bd-49b4-aecb-bd61e70d5fcd)
But it would be great if it supports tabIndex so I can just pass them normally.
